package.json

```diff
@@ -14,6 +14,6 @@
     "sass": "^1.69.5"
   },
   "dependencies": {
-    "govuk-frontend": "^4.7.0"
+    "govuk-frontend": "4.7.0"
```

Comment:
could you revert this change too

Reply:
I'd argue we should pinning exact versions and letting Dependabot track package updates as it does already

https://github.com/ministryofjustice/data-platform-control-panel/blob/13d86aaa658d3e560833ffc6df1d2b0c7e4ddc4a/.github/dependabot.yml#L20-L23